src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -57,16 +57,25 @@ namespace chip {
 
 inline constexpr size_t kMaxDeviceTransportBlePendingPackets = 1;
 
-using DeviceTransportMgr = TransportMgr<Transport::UDP /* IPv6 */
+inline constexpr size_t kMaxDeviceTransportTcpActiveConnectionCount = CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_TCP_CONNECTIONS;
+
+inline constexpr size_t kMaxDeviceTransportTcpPendingPackets = CHIP_CONFIG_MAX_TCP_PENDING_PACKETS;
+
+using DeviceTransportMgr =
+    TransportMgr<Transport::UDP /* IPv6 */
 #if INET_CONFIG_ENABLE_IPV4
-                                        ,
-                                        Transport::UDP /* IPv4 */
+                 ,
+                 Transport::UDP /* IPv4 */
```

Comment:
Why specify 'Transport::UDP' twice for IPv4 and IPv6? Does TCP only supports IPv6?

Reply:
The UDP for IPv4 is legacy and I kept it as-is. For TCP, I have only defined it for IPv6. We need to look into if the UDP over IPv4 is required or not.